L73-java-binderchannel/wireformat.md

```diff
@@ -50,26 +50,32 @@ it will shutdown the transport gracefully.
 #### Transaction Data
 
 ```
-major version = int;
+version = int;
 protocol extension flags = int;
 num bytes = int;
 ping id = int;
 
-setup transport transaction = major version, binder, [protocol extension flags];
+setup transport transaction = version, binder, [protocol extension flags];
 shutdown transport transaction = nothing;
 acknowledge bytes transaction = num bytes;
 ping transaction = ping id;
 ping response transaction = ping id;
 ```
 
-During transport setup, if a server receves a setup transaction with a higher
-protocol version, it should respond with the protocol version _it_ supports.
+==setup transport transaction== is guaranteed to be backward-compatible and so
```

Comment:
Did you mean to use backtick or `**` here? `==` doesn't format into anything here.

Reply:
Oops, fixed.